spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -112,6 +115,28 @@
               )
               allow(controller).to receive(:pii_requested_but_locked?).and_return(false)
               action
+
+              sp_return_log = SpReturnLog.find_by(issuer: client_id)
+                expect(sp_return_log.ial).to eq(2)
+            end
+
+            context 'with the IAL2-string flag' do
```

Comment:
**question:** is this supposed to be `IAL2-strict`?
```suggestion
            context 'with the IAL2-strict flag' do
```

Reply:
yep